README.md

```diff
@@ -276,9 +276,9 @@ _Environment variables precedence order is as follows. Top to bottom, top entry
 
 Meaning that, environment variables override defaults in the configuration file but are superseded by options to `df.spawn({...})`
 
-#### IPFS_JS_EXEC and IPFS_GO_EXEC
+#### IPFS_JS_EXEC, IPFS_GO_EXEC, and IPFS_RUST_EXEC
```

Comment:
@hugomrdias Are these still in play? We use them in our `ipfs-rust-conformance` testing and also see them in `interop` but we're hard pressed to find them in `js-ipfsd-ctl` or any `npm-**-ipfs-dep` repos

Reply:
Yes, go-ipfs uses them [in CI](https://github.com/ipfs/go-ipfs/blob/master/.circleci/config.yml#L207)